src/Symfony/Component/Messenger/Transport/Semaphore/SemaphoreStamp.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Semaphore;
+
+use Symfony\Component\Messenger\Stamp\NonSendableStampInterface;
+
+/**
+ * @author Cedrick Oka Baidai <okacedrick@gmail.com>
+ */
+class SemaphoreStamp implements NonSendableStampInterface
+{
+    /**
+     * @var int
```

Comment:
Such docblocks on properties can be removed as soon as the type is already identified by the constructor typehints & assigments